docs/DeveloperGuide.md

```diff
@@ -426,6 +438,95 @@ In the diagram, the predicates `modulePredicate` and `taskPredicate` are the cus
 
 <img src="images/CdSequenceDiagram.png" width="1000" />
 
+### Marking modules as done
+
+Section by : [Marciano](https://github.com/midnightfeverrr)
+
+Marking modules as done allows users to selectively hide modules that they have completed.
+
+Every module in the module list will be in either the done or undone state.
+New tag/label will be created to mark a module as `done`.
```

Comment:
Could just simplify to either tag or label, rather than both, to avoid ambiguity, but not critical